jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/overview/CaseOverviewViewImpl.java

```diff
@@ -162,6 +158,11 @@ public void setCaseId(final String description) {
         caseId.setTextContent(description);
     }
 
+    @Override
+    public void setCaseOwner(final String owner) {
+        caseOwner.setTextContent(" (" + owner + ")");
```

Comment:
Perhaps the brackets are more a visual thing, moving it to the html template would make sense to me.